src/createLoggerMiddleware.js

```diff
@@ -34,7 +34,7 @@ export default ({logger}) => {
         if (req.method === 'POST' && req.body && req.body.operationName) {
           meta.graphql = {
             operationName: req.body.operationName,
-            variables: req.body.variables
+            ...(logGraphqlVariables ? {variables: req.body.variables} : null)
```

Comment:
Are we fine with no GraphQL variables being logged anymore in general when an app turns this off?

Alternatively, we could also use a heuristic to exclude large payloads, e.g.:

```js
const isTooLarge = JSON.stringify(req.body.variables).length > THRESHOLD;
```

That way we could keep the variables for other cases (e.g. to know which article has been fetched as part of an `Article` operation).

Just as a suggestion though, have you thought more about this?

Reply:
Actually that is a great idea - added a new commit for this change.